core/src/main/scala/kafka/server/BrokerToControllerRequestManager.scala

```diff
@@ -63,6 +64,18 @@ abstract class AbstractBrokerToControllerRequestManager[Item <: BrokerToControll
   // Used to allow only one in-flight request at a time
   private val inflightRequest: AtomicBoolean = new AtomicBoolean(false)
 
+  // Additional observability into the internal states
+  private val baseMetricTags = Map("class" -> getClass.getName)
```

Comment:
I'm wondering if this work "private val baseMetricTags = ("class", getClass.getName)"

Reply:
1. Map is more idiomatic in Scala.  And the way you do it is making it a tuple (more of Java style)
2. The signature of metric creation below requires a map.

Reply:
Sounds good. Thanks for the explanation.